docs/guide/getting-started.html

```diff
@@ -3,7 +3,6 @@
 
   <html>
     <head>
-      <title>Getting Started</title>
```

Comment:
All the titles are gone, is that on purpose? (I honestly don't know how the documentation works, this just caught my eye)